sources/packages/web/src/components/generic/StatusBadge.vue

```diff
@@ -5,7 +5,9 @@ status Badge
   <v-badge :color="backGroundColor" :text-color="textColor" :class="badgeClass">
     <template v-slot:badge>
       <font-awesome-icon icon="circle" class="mr-1" :color="iconColor" />
-      <span class="text-uppercase">{{ status }}</span>
+      <span class="text-uppercase">{{
+        status === GeneralStatusForBadge.ActiveRestriction ? "active" : status
```

Comment:
may this manipulation we can do it in the `setup`, by creating a new const called `label` and assigning value to label in function `setStyles`

Reply:
Feedback comment addressed.